server/lib/nl/detection/place.py

```diff
@@ -275,6 +276,16 @@ def get_place_from_dcids(place_dcids: List[str], debug_logs: Dict) -> any:
                              place_type=ptype,
                              country=country)
 
+  # If there were detected places that there wasn't any place info for, assume
+  # they are non-place entities and get info through property values.
+  missing_place_info = [p for p in place_dcids if not dcid2place.get(p)]
```

Comment:
Should the determination of place vs. entity be based on the returned type?  (available in Mixer csv)